cmd/terramate/cli/cli.go

```diff
@@ -875,17 +873,22 @@ func (c *cli) initAnalytics(cmd string, opts ...tel.MessageOpt) {
 	anasigfile := filepath.Join(c.clicfg.UserTerramateDir, "analytics_signature")
 	credfile := filepath.Join(c.clicfg.UserTerramateDir, credfile)
 
+	var repo *git.Repository
+	if c.prj.isRepo {
+		repo, _ = c.prj.repo()
```

Comment:
_nitpick_ it would be better to have a function in the type to return `repo, isRepo, err := c.prj.repo()`?

Reply:
I agree, I don't like the fatal() in `repo()`, but at the same time I do not want to refactor this at this time.